src/App.js

```diff
@@ -2,15 +2,61 @@ import React from 'react';
 import './App.scss';
 
 class App extends React.Component {
-  addOne = () => {};
+  state = {
+    counter: 0,
+  }
+
+  addOne = () => {
+    this.setState(state => ({
+      counter: state.counter + 1,
+    }));
+  }
 
-  add100 = () => {};
+  add100 = () => {
+    this.setState(state => ({
+      counter: state.counter + 100,
+    }));
+  };
 
-  increase = () => {};
+  increase = () => {
+    const { counter } = this.state;
+
+    if (counter % 5 === 0) {
+      this.add100();
+    } else {
+      this.addOne();
+    }
+  };
```

Comment:
за умовою задачі там трошки по-іншому
```suggestion
  increase = () => {
    const { counter } = this.state;

    if (counter % 5 === 0) {
      this.add100();
    }
      
    this.addOne();
    
  };
```